src/main/java/ro/unibuc/triplea/application/games/steam/dto/response/SteamGameStandardResponse.java

```diff
@@ -0,0 +1,14 @@
+package ro.unibuc.triplea.application.games.steam.dto.response;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+public class SteamGameStandardResponse {
+    private String code;
+    private String message;
+    private Object data;
```

Comment:
Why data is type of Object? object is an internal class and should not be used

Reply:
Using ```Object``` allows flexibility in the type of data that can be stored in the ```data``` field, even though it sacrifices type safety. I took as model the ```StandardResponse``` class in auth implementation, and made a specific one for the Steam Game package. I modified the type now to String, but I think hanges should be made in auth package too.